.drone.yml

```diff
@@ -0,0 +1,51 @@
+kind: pipeline
+name: default
+
+platform:
+    os: linux
+    arch: amd64
+
+steps:
+- name: edelweiss_swc
+  image: mythx.azurecr.io/tools/edelweiss
+  environment:
+    IGNORE_FALSE_POSITIVES:
+      from_secret: IGNORE_FALSE_POSITIVES
+    IGNORE_REGRESSIONS:
+      from_secret: IGNORE_REGRESSIONS
+    AWS_DEFAULT_REGION:
+        from_secret: AWS_DEFAULT_REGION
+    AWS_ACCESS_KEY_ID:
+        from_secret: AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY:
+        from_secret: AWS_SECRET_ACCESS_KEY
+    MONGO_URL:
+        from_secret: MONGO_URL
+  commands:
+    - export CIRCLE_BUILD_NUM=$DRONE_BUILD_NUMBER
+    - export CIRCLE_BUILD_URL=$DRONE_BUILD_LINK
+    - export CIRCLE_BRANCH=$DRONE_BRANCH
+    - export CIRCLE_SHA1=$DRONE_COMMIT_SHA
+    - cd /edelweiss
+    - git submodule update --init --recursive
+    - git submodule update --remote --recursive
+    - pip3 install mythril
```

Comment:
won't this install the mythril ver sion from the package repo?

Reply:
changed to install from local path